libs/core/src/lib/metadata/configuration.ts

```diff
@@ -2,22 +2,32 @@ import { Type } from '@angular/core';
 
 import { createMetadataGetSet } from './util';
 
-export interface ConfigurationMeta {
+const Reflect = (window as any).Reflect;
```

Comment:
User should be able to provide Reflect API differently otherwise default back to getting it from global window.